pepper-apis/src/main/java/org/broadinstitute/ddp/model/activity/definition/question/PicklistOptionDef.java

```diff
@@ -33,6 +36,12 @@
     @SerializedName("exclusive")
     private boolean isExclusive;
 
+    @SerializedName("nestedOptionsLabelTemplate")
+    private Template nestedOptionsLabelTemplate;
+
+    @SerializedName("nestedPicklistOptions")
```

Comment:
Can we simply call this `nestedOptions`?

Reply:
@fat23cat 
Pavel: Made related changes to your front end branch: https://github.com/broadinstitute/ddp-angular/pull/335/commits/574ecc2d56f0add141c69b562365ea17c074127d
cc @MocanaAtBroad